test/parallel_api/experimental/radix_sort_esimd.pass.cpp

```diff
@@ -296,23 +315,32 @@ int main()
         {
             test_general_cases<uint32_t>(size);
             test_general_cases<int>(size);
+            test_general_cases<int16_t>(size);
+            test_general_cases<uint16_t>(size);
             test_general_cases<float>(size);
             // test_general_cases<double>(size);
         }
         for(auto size: coop_sizes)
         {
             test_general_cases<uint32_t>(size);
             test_general_cases<int>(size);
+            test_general_cases<int16_t>(size);
+            test_general_cases<uint16_t>(size);
             test_general_cases<float>(size);
             // test_general_cases<double>(size);
         }
         for(auto size: onesweep_sizes)
         {
             test_usm<uint32_t>(size);
             test_usm<int>(size);
+            test_usm<int16_t>(size);
+            test_usm<uint16_t>(size);
             test_usm<float>(size);
         }
-        test_small_sizes();
+        test_small_sizes<uint32_t>();
+        test_small_sizes<int>();
+        test_small_sizes<int16_t>();
+        test_small_sizes<uint16_t>();
```

Comment:
Let's leave only on case as before. The purpose of that test case is to make sure the input consequence is not changed when the number of elements to sort is 0 or 1. There is no need to test different types.

Reply:
Fixed